packages/nest/src/oauth/oauth.sql.provider.ts

```diff
@@ -210,25 +212,29 @@ export class OAuthSQLProvider implements OAuthProvider {
         );
     }
 
-    private userInfoRepository(): RepositoryMapper<OAuthUserInfoEntity, OAuthUserInfoModel> {
+    private userInfoDataSource(): DataSourceMapper<RawSQLDataSource, OAuthUserInfoEntity> {
```

Comment:
I agree, but as other comments this is out of the scope of this PR.

In another project I've created a `@Cached` decorator for this sole purpose, so that we can use methods to have the code better organized but at the same time without extra instantiation. Maybe we can add it to `core`.

Follow up issue created: https://github.com/mobilejazz/harmony-typescript/issues/103

@orioljp resolve this conversation if you think we can continue on the issue (update the issue with missing information).

Reply:
Oh no, it was me, I split the repository in two… lazy madafaka. xD

Reply:
Fixed in b38b4b9ae0297b686a440f1134220b6879ebf4f1